difflint/lib.py

```diff
@@ -13,6 +14,29 @@
 MISSING_FILE_EXIT_CODE = 72  # os.EX_OSFILE is not portable
 
 
+def make_path_from_cwd(*args):
+    """
+    Builds a string containing the current working directory as the root
+    and appends zero or more strings to the end representing pieces of a
+    filepath. This is cross-platform but the output will vary depending
+    on where the current working directory is.
+
+    Inputs:
+        *args: Zero or more strings comprising the suffix of a filepath,
+               excluding seperators such as forward or backslashes.
+
+    Output: A platform aware filepath string with the current working
+            directory as the root and any suffix arugments at the end.
+
+    Example:
+        (Run from the directory ~/my_repo on a Linux system.)
+
+        > make_path_from_cwd("math", "is", "fun.txt")
+        > /Users/devinekins/my_repo/math/is/fun.txt
+    """
+    return getcwd() + os.path.join(os.path.sep, *args)
```

Comment:
I don't think `os.path.join` is meant to be used with path separators, it should put in its own separators. Seems to me this should be `os.path.join(getcwd(), *args)`